test/ng/directive/ngRepeatSpec.js

```diff
@@ -1141,7 +1143,7 @@ describe('ngRepeat', function() {
       scope.$digest();
       expect($exceptionHandler.errors.shift().message).
           toMatch(
-            /^\[ngRepeat:dupes\] Duplicates in a repeater are not allowed\. Use 'track by' expression to specify unique keys\. Repeater: item in items, Duplicate key: object:3, Duplicate value: {}/);
+            /^\[ngRepeat:dupes] Duplicates in a repeater are not allowed\. Use 'track by' expression to specify unique keys\. Repeater: item in items, Duplicate key: object:3, Duplicate value: /);
```

Comment:
>
> In test/ng/directive/ngRepeatSpec.js
> <https://github.com/angular/angular.js/pull/14952#discussion_r72969231>:
>
> > @@ -1141,7 +1143,7 @@ describe('ngRepeat', function() {
> >        scope.$digest();
> >        expect($exceptionHandler.errors.shift().message).
> >            toMatch(
> > -            /^\[ngRepeat:dupes\] Duplicates in a repeater are not allowed\. Use 'track by' expression to specify unique keys\. Repeater: item in items, Duplicate key: object:3, Duplicate value: {}/);
> > +            /^\[ngRepeat:dupes] Duplicates in a repeater are not allowed\. Use 'track by' expression to specify unique keys\. Repeater: item in items, Duplicate key: object:3, Duplicate value: /);
>
> Why remove the {} at the end (here and below)?
>
Because they weren't doing anything, it were just empty patterns which is
why ESLint was complaining about them.